tests/create_cluster.sh

```diff
@@ -5,7 +5,7 @@
 k8s_version=$1
 
 if [[ $IS_LOCAL == "false" ]]; then
-  sudo -E minikube start --vm-driver=none --cpus 4 --memory 4096 --kubernetes-version=$k8s_version &> $HOME/minikube.log 2>&1 < /dev/null
+  sudo -E minikube start --vm-driver=none --cpus 6 --memory 6144 --kubernetes-version=$k8s_version &> $HOME/minikube.log 2>&1 < /dev/null
```

Comment:
Is the increment in resources due to some new dependency?

Reply:
Increased demands for the tests with added GRPC clients. I noticed the tests were flaky and seemed to be failing to timeout. Experience with running minikube has shown that the lack of responsiveness occurs when it becomes resource strapped so I increased it to account for this. I also had to increase the integration test vm resources to account for this change as well.